README.md

```diff
@@ -2,8 +2,27 @@
 
 This repository is a work in progress for running a TIMS terminology server. This is a demo setup for being able to simply clone this repo to your machine and then run:
 
-* bin/start-ts.sh  # to start the server
-* bin/stop-ts.sh  # to stop it.
-* bin/reset-ts.sh  # to rest to a clean checkout to start over.
+* `bin/start-ts.sh`  # to start the server
+* `bin/stop-ts.sh`  # to stop it.
+* `bin/reset-ts.sh`  # to rest to a clean checkout to start over.
 
 See [this doc](https://docs.google.com/document/d/1J7exF73Z0Z8W-SCYRrQ5qNQVMZTkIH0c4rf-sgjDBfk) for work in progress documentation and notes.
+
+## Prerequisites
+1. Java 11+
+2. Python 3.8+
+3. Docker
+
+## Server management
+### Starting the server
```

Comment:
New section. This is where I added how to access the server when it starts. Figured this should be front and center,  not just in the google doc.

Reply:
removed and added to docs pr